tap_mssql/sync_strategies/common.py

```diff
@@ -84,12 +84,7 @@ def generate_select_sql(catalog_entry, columns):
     escaped_columns = [escape(c) for c in columns]
 
     select_sql = "SELECT {} FROM {}.{}".format(",".join(escaped_columns), escaped_db, escaped_table)
-
-    # escape percent signs
-    select_sql = select_sql.replace("%", "%%")
-    return select_sql
-
-
+    
```

Comment:
you got rid of the return statement, line 90.  That should be kept.

Reply:
@wmattern0 good catch